src/components/menus/directory-tree-contextual-menu.tsx

```diff
@@ -91,6 +91,7 @@ export default function DirectoryTreeContextualMenu(props: Readonly<DirectoryTre
     );
 
     const [renameCB, renameState] = useDeferredFetch(
+        // @ts-ignore
```

Comment:
you can do `// @ts-expect-error refacto typing of useDefferedFetch`

Reply:
Done